deepchecks/vision/checks/distribution/image_property_drift.py

```diff
@@ -123,28 +139,33 @@ def compute(self, context: Context) -> CheckResult:
             value: dictionary containing drift score for each image property.
             display: distribution graph for each image property.
         """
+        assert self._train_properties is not None
```

Comment:
This should return a meaningful Deepchecks error in case it isn't true (such as "No samples belonging to classes...")

Reply:
this is totally internal stuff, and it could happen only if you forgot to initialize `self._train_properties` within `initialize_run`.

I added this to stop the static type checker from complaining that type of the `self._train_properties` is `Optional[Dict[...]]`